<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MostrarReservas</class>
 <widget class="QMainWindow" name="MostrarReservas">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>601</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>659</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Mostrar Reservas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Recursos/img/bg.qrc">
    <normaloff>:/prefijoNuevo/brianda1.png</normaloff>:/prefijoNuevo/brianda1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Estilo general para la ventana */
QWidget {
	background-color: rgba(38, 45, 67, 100);
    color: #E0E0E0; /* Texto claro */
    font-family: 'Segoe UI', sans-serif;
    font-size: 9pt;
}

/* Estilo para el QListWidget */
QListWidget {
    background-color: #34495E; /* Fondo oscuro */
    border: 1px solid #555555;
    border-radius: 5px;
    color: #E0E0E0; /* Texto claro */
    padding: 5px;
}

/* Estilo para el QTableWidget */
QTableWidget {
    background-color: #34495E; /* Fondo oscuro */
    border: 1px solid #555555;
    border-radius: 5px;
    gridline-color: #555555; /* Color de las líneas de la cuadrícula */
    color: #E0E0E0; /* Texto claro */
}


QTableWidget::item {
    border: 1px solid #555555;
}


/* Estilo para los botones */
QPushButton {
    background-color: #2C3E50; /* Fondo oscuro */
    color: #E0E0E0; /* Texto claro */
    border: 1px solid #555555;
    border-radius: 5px;
    padding: 5px 10px;
    font-size: 11pt;
}

QPushButton:hover {
    background-color: #34495E; /* Fondo más claro al pasar el ratón */
}

QPushButton:pressed {
    background-color: #1C2A36; /* Fondo más oscuro al presionar */
}


/* Estilo para el menú */
QMenuBar {
    background-color: #2C3E50; /* Fondo oscuro */
    color: #E0E0E0; /* Texto claro */
}

QMenuBar::item {
    background-color: transparent;
    padding: 5px 10px;
}

QMenuBar::item:selected {
    background-color: #34495E; /* Fondo para elementos seleccionados */
}

QMenu {
    background-color: #2C3E50; /* Fondo oscuro */
    color: #E0E0E0; /* Texto claro */
    border: 1px solid #555555;
}

QMenu::item:selected {
    background-color: #34495E; /* Fondo para elementos seleccionados */
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="asg_botonReservar">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>490</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Acceder a la pantalla de Reservas</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Reservar</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="asg_edit_reserva">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Acceder a la pantalla de Reservas para editar</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="asg_consultar">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Acceder a la pantalla de Reservas para editar</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Consultar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="asg_eliminar">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Acceder a la pantalla de Reservas para editar</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>681</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="asg_listaWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>104</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>156</width>
         <height>492</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">    background-color: #2C3E50; /* Fondo oscuro */</string>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTableWidget" name="asg_tablaWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>190</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>554</width>
         <height>492</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: #2C3E50; /* Fondo oscuro */

}
    QHeaderView::section {
        background-color:  rgb(20, 24, 36);
    }</string>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::TextElideMode::ElideLeft</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollMode::ScrollPerItem</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollMode::ScrollPerItem</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::PenStyle::NoPen</enum>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>5</number>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>32</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>122</number>
       </attribute>
       <column>
        <property name="text">
         <string>idReserva</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fecha</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Persona</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Teléfono</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Tipo de Reserva</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Recursos/img/bg.qrc"/>
 </resources>
 <connections/>
</ui>
